examples/example_notebooks/stable_baselines3_example.ipynb

```diff
@@ -0,0 +1,456 @@
+    "            batch_size=batch_size, gamma=gamma, policy_kwargs=policy_kwargs, \n",
+    "            exploration_fraction=exploration_fraction, target_update_interval=target_update_interval,\n",
+    "            verbose=verbose)\n",
+    "model.learn(total_timesteps=nb_steps)\n",
+    "env.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.4 Saving the model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When the training has finished you can save the model your DQN has learned to reuse it later, e.g. for evaluation or if you want to continue your training. For this, each Stable Baselines3 algorithm has a .save() function where you only have to specify your path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.save(Path.cwd() / \"saved_agents\" / \"TutorialAgent\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Evaluating an agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After you have trained your agent you would like to see how well it does on your control problem. For this you can look at a visual representation of your currents in a test trajectory or see how well your agent does in a test scenario."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.1 Loading a model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, before you start your evaluation you have to load a trained agent or take the trained agent from above which is still saved in the variable model. To load a trained agent you simply have to call the .load() function of your algorithm with the respective path. If you have executed the provided code above you can uncomment the first line of code and comment the second one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#model = DQN.load(Path(__file__).parent / \"saved_agents\" / \"TutorialAgent\")  #your agent\n",
+    "model = DQN.load(Path.cwd() / \"saved_agents\" / \"TutorialPreTrainedAgent\") #pre-trained agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.2 Taking a look at the mean reward per episode during training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The RewardLogger callback saved the mean reward per episode during training. There you can observe, if the training reward did grow over time. Uncomment the first line and comment the second line if you have executed the training cell. An exemplary graph could look like this:\n",
+    "\n",
+    "![TrainingRew](img/TrainingRewEpisode.png)"
```

Comment:
This seems to appear magically from nowhere.
Please write a small code snippet visualizing the reward over time appropriately.